modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/project/ProjectResource.java

```diff
@@ -184,6 +186,14 @@ public ProjectJson get( final @QueryParam("name") String projectNameValue )
         return doCreateJson( this.projectService.get( projectName ) );
     }
 
+    @GET
+    @Path("getTree")
```

Comment:
I suggest to use `tree` instead off `getTree`, because `get` present in HTTP method

Reply:
If we have `list` this one should be called `tree`, I agree